components/service/glean/src/main/java/mozilla/components/service/glean/private/MetricAliases.kt

```diff
@@ -22,11 +24,11 @@ typealias MemoryDistributionMetricType = mozilla.telemetry.glean.private.MemoryD
 typealias MemoryUnit = mozilla.telemetry.glean.private.MemoryUnit
 typealias PingType<T> = mozilla.telemetry.glean.private.PingType<T>
 typealias QuantityMetricType = mozilla.telemetry.glean.private.QuantityMetricType
-typealias RecordedExperimentData = mozilla.telemetry.glean.private.RecordedExperimentData
+typealias RecordedExperiment = mozilla.telemetry.glean.private.RecordedExperiment
```

Comment:
Is this a breaking change we need to call out in A-C's changelog?

Reply:
Technically yeah, I will call it out.
Though it doesn't break any consumers right now, because they don't use that type name anywhere.